wp-content/themes/currentorg/inc/business-directory-plugin.php

```diff
@@ -296,6 +296,42 @@ function wpbdp_single_listing_page_template( $page_template ) {
 }
 add_filter( 'page_template', 'wpbdp_single_listing_page_template' );
 
+/**
+ * Output flavor text and a list of tags on the category page
+ *
+ * This is for a hook in /plugins/business-directory-plugin/templates/page.tpl.php
+ *
+ * @param string $context The WPBDP page child ID, where "ID" is "category", "main_page", "all_listings",  or something like that
```

Comment:
is `$context` still used in this function?

Reply:
Nope.